src/core/Akka.Cluster/Configuration/Cluster.conf

```diff
@@ -169,6 +169,8 @@ akka {
     # greater than this value.
     reduce-gossip-different-view-probability = 400
 
+    use-legacy-heartbeat-message = false
```

Comment:
Probably should add a comment here, but we can do that in a future PR

Reply:
Done